### DISCUSSION TITLE
Display the `Name` of a LogEvent in the logs of `TransactionReceipt`

### DISCUSSION THREAD

**Discussion by magiclars-off** [wevm/viem#642] [General]
I'm looking for a way to extract a `Transfer` event from a tx in the past (so not watching a current contract).

I'm working with a `TransactionReceipt` which doesn't seem to contain the name

![image](https://github.com/wagmi-dev/viem/assets/62888804/9d2ac583-6215-486d-8ab5-f24edf924b3a)

![image](https://github.com/wagmi-dev/viem/assets/62888804/bf483568-5a43-445e-8fca-f59fc7eff287)

**Comment by jxom** [comment]
You can use https://viem.sh/docs/contract/decodeEventLog.html!
